README.rst

```diff
@@ -47,14 +47,12 @@ For other options see ``linkchecker --help``.
 Docker usage
 -------------
 
-*The Docker images are out-of-date, pip installation is the only currently recommended method.*
-
 If you do not want to install any additional libraries/dependencies you can use the Docker image.
 
 Example for external web site check::
 
-  docker run --rm -it -u $(id -u):$(id -g) linkchecker/linkchecker --verbose https://www.example.com
+  docker run --rm -it -u $(id -u):$(id -g) ghcr.io/linkchecker/linkchecker:master --verbose https://www.example.com
```

Comment:
Using `master` or `latest` is not a good practice.
If you do the proper tagging of container images then you should use something like:
`ghcr.io/linkchecker/linkchecker:10.0.1` (or just `linkchecker:10`)

Reply:
Well there are multiple use cases and different sorts of users.

For the rolling image we could use 10, but it would be a bit of an illusion, 9 came out in 2014. Updated in edc974e, to latest provided by docker/metadata-action.

But fortunately we can have multiple tags so some choice now after that commit - semvar tags for application releases and git commit checksums as well.

Also updated the version the application reports to include the git commit checksum if post a release.